content/en/monitors/service_level_objectives/monitor.md

```diff
@@ -51,6 +51,16 @@ an SLO breach.
 Here we add contextual information about the purpose of the SLO, including any related information
 in the description and tags you would like to associate with the SLO.
 
+#### Underlying monitor and SLO histories
+
+Making changes to the monitor used by an SLO recalculates the SLO history. Therefore, the monitor history and SLO history may not match after a monitor update.
+
+Datadog recommends against using monitors with `Alert Recovery Threshold` and `Warning Recovery Threshold` as they can also affect your SLO calculations and do not allow you to cleanly differentiate between a SLI's good behavior and bad behavior.
+
+SLO calculations do not take into account when a monitor is resolved manually or as a result of the **_After x hours automatically resolve this monitor from a triggered state_** setting. If these are important tools for your workflow, consider cloning your monitor, removing auto-resolve settings and `@-notification`s, and using the clone for your SLO.
+
+Confirm you are using the preferred SLI type for your use case. Datadog supports monitor-based SLIs as described here. You can also use metric-based SLIs as [described in the SLO event documentation][5].
```

Comment:
`Datadog supports monitor-based SLIs as described here`

To which page the here should point to? is it the same as the `[5]` reference?

Reply:
ping @unclebconnor

Reply:
@l0k0ms  Yep that's the one.

Reply:
```suggestion
Confirm you are using the preferred SLI type for your use case. Datadog supports monitor-based SLIs and metric-based SLIs as [described in the SLO event documentation][5].
```